awx/main/management/commands/reload_awx_plugins.py

```
@@ -0,0 +1,14 @@
+from awx.main.models import CredentialType
+
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = 'Reload CredentialTypes to pull new ones into existing AWX installs.'
+
+    def handle(self, *args, **options):
```

Ruff (0.15.1)

[warning] Unused method argument: `args`

(ARG002)